README.md

```diff
@@ -81,3 +81,7 @@ The website is built using **Next.js**, a React-based framework for server-side
 
 5. Look into .env.local.example file, it should have the necessary environmental variables needed for this project.
 
+## Testing
+
+1. 
```

Comment:
running `yarn test` worked for me, but it's not clear if the intention is for running unit test on the front-end, back-end, or running integration tests. Also just because we've been talking about coverage, I think it would be nice to note how to run that. I found `yarn test --coverage` works, but it's not clear the scope of the coverage.

Reply:
woops! Yes, i'll add that note to Readme.  I'll add --coverage to the script too, so when you run yarn test it should show the coverage stats.